config/stack/manifests/resources/yugabytecluster.resource.yaml

```diff
@@ -0,0 +1,23 @@
+id: yugabytecluster
+title: YugabyteDB Cluster
+titlePlural: YugabyteDB Clusters
+category: Database
+overviewShort: "A YugabyteCluster configures a Rook 'YugabyteCluster'"
```

Comment:
according to https://rook.io/docs/rook/v1.1/yugabytedb-cluster-crd.html, the Rook CRD kind is actually `YBCluster`.  If we want to use the fully qualified name of the type, it's `ybclusters.yugabytedb.rook.io`